RELEASE.md

```diff
@@ -0,0 +1,221 @@
+    If your account does not have permissions, then you'll need to ask one of
+    the [organization administrators](https://bintray.com/google) to give you
+    permissions to update the `tensorflow` repository. Please keep the
+    [repository option](https://bintray.com/google/tensorflow/edit?tab=general)
+    to *"GPG sign uploaded files using Bintray's public/private key pair"*
+    **unchecked**, otherwise it will conflict with locally signed artifacts.
+-   A GPG signing key, required
+    [to sign the release artifacts](http://central.sonatype.org/pages/apache-maven.html#gpg-signed-components).
+
+### Preparing a release
+
+#### Major or minor release
+
+1.  Get a clean version of the source code by cloning the
+    [TensorFlow Java GitHub repository](https://github.com/tensorflow-java)
+    ```
+    git clone https://github.com/tensorflow/java
+    ```
+2.  Create a new branch for the release named `r<MajorVersion>.<MinorVersion>`
+    ```
+    git checkout -b r1.0
+    ```
+3.  Update the version of the Maven artifacts to the full version of the release
+    ```
+    mvn versions:set -DnewVersion=1.0.0
+    ``` 
+4.  Commit the changes and push the new branch to the GitHub repository
+    ```
+    git add .
+    git commit -m "Releasing 1.0.0"
+    git push --set-upstream upstream r1.0
+    ```
+
+#### Patch release
+
+1.  Get a clean version of the source code by cloning the
+    [TensorFlow Java GitHub repository](https://github.com/tensorflow-java)
+    ```
+    git clone https://github.com/tensorflow/java
+    ```
+2.  Switch to the release branch of the version to patch
+    ```
+    git checkout r1.0
+    ```
+3.  Patch the code with your changes. For example, changes could be merged from another branch you
+    were working on or be applied directly to this branch when the required changes are minimal.
+
+4.  Update the version of the Maven artifacts to the full version of the release
+    ```
+    mvn versions:set -DnewVersion=1.0.1
+    ``` 
+4.  Commit the changes and push the branch to the GitHub repository
+    ```
+    git add .
+    git commit -m "Releasing 1.0.1"
+    git push
+    ```
+
+### Building native artifacts
+
+Once a release branch (i.e. a branch prefixed by `r`) is pushed on the repository, a workflow will
+automatically start in GitHub Actions to build the native artifacts for all supported
+architures/platforms and deploy them temporarily on OSSRH for staging.
+
+There is no user action required for this step other than watching the progress of the GitHub
+Actions workflow and making sure that all steps have been completed successfully.
+
+#### Build native artifacts manually
+
+Some platforms cannot be build successfully on GitHub Actions, due to some limits to their resources
+(e.g. max 6 hours for a job). For this reasons, we need to build manually some of our artifacts on
+private servers. 
+
+To do so, follow the same steps as the [CI build](https://github.com/tensorflow/java/.github/workflows/ci.yml) 
```

Comment:
This link should be something like this - https://github.com/tensorflow/java/blob/master/.github/workflows/ci.yml

Reply:
Wow, I wrote that thing to fast, all broken links are new ones :-/